ls1intum/Artemis
# [Bugfix] Import a prog. exercise by cloning instead of forking

### Checklist

### Motivation and Context


@kloessst found an issue (#2451) when importing programming exercises more than 5 times. Bitbucket does not allow forking for more than 5 times of the same root repository.

### Description

We solve this issue by not forking at all but rather pushing the content of a source repository into a target's repository. 
Closes #2451 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new programming exercise A
4. Import this exercise and name it A1
5. Import exercise A1 and name it A2. 
6. Repeat step 4 and 5 five times until you get the exercises A1-A5
7. Import the exercise A5 and verify that the sixth importing still succeeds

### Test Coverage

BitbucketService.java: 76%
GitService.java: 79%
